docs/site/package.json

```diff
@@ -13,20 +13,23 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "^8.1.0",
+    "@angular/cdk": "^8.2.3",
     "@angular/common": "^8.1.0",
     "@angular/compiler": "^8.1.0",
     "@angular/core": "^8.1.0",
     "@angular/elements": "^8.1.1",
-    "@angular/flex-layout": "^8.0.0-beta.26",
+    "@angular/flex-layout": "^8.0.0-beta.27",
     "@angular/forms": "^8.1.0",
-    "@angular/material": "^8.1.0",
+    "@angular/material": "^8.2.3",
     "@angular/platform-browser": "^8.1.0",
     "@angular/platform-browser-dynamic": "^8.1.0",
     "@angular/pwa": "^0.801.1",
     "@angular/router": "^8.1.0",
     "@angular/service-worker": "^8.1.0",
+    "@bazel/bazel": "^1.2.0",
```

Comment:
You don't need to depend on bazel here. You can invoke bezel commands from workspace root.